irohad/main/irohad.cpp

```diff
@@ -139,6 +145,22 @@ int main(int argc, char *argv[]) {
       return EXIT_FAILURE;
     }
 
+    // check if genesis block already existing
+    auto genesis_block_exist = false;
+    irohad.storage->getBlockQuery()->getTopBlocks(1).subscribe(
+        [&genesis_block_exist](auto block) { genesis_block_exist = true; });
+
+    // Check if force flag to overwrite genesis block specified
+    if (genesis_block_exist) {
+        if (not FLAGS_genesis_block) {
+            log->error("Genesis block already exists in transaction directory. Use
```

Comment:
Maybe `Block store is not empty` will express the case more generally than `Genesis block already exists in transaction directory`.

Reply:
Damn ! Sorry for these silly mistakes, crazy week at work and had to do it in a kind of hurry. Apologies. I will fix them ASAP.